src/procedures/procedures.11tydata.yaml

```diff
@@ -555,6 +555,12 @@ procedureDisableMultiProtocol:
         <ol>
```

Comment:
(Looks like I can't leave review comments on unedited lines)

In the step above, the title is "Disable SkyConnect Multi-PAN". Can we change it to match the OTBR wording: "Uninstall the Silicon Labs Multiprotocol add-on"?